src/Dependency/TagDependency.php

```diff
@@ -141,9 +142,11 @@ private static function buildCacheKeys($tags): array
      * @param CacheInterface $cache
      *
      * @return array
+     * @psalm-return array<array-key, string|null>
```

Comment:
Why values is `string|null`?
`CacheInterface::getMultiple()` returned `array<array-key, mixed>`.

Reply:
Because of the line [73](https://github.com/yiisoft/cache/blob/master/src/Dependency/TagDependency.php#L73) tag data fill with microtome() if the tag is not null